### PR TITLE
Timeline adjustments

### PR DESCRIPTION
**Pull request by noirbizarre**
Little adjustments and fixes on timeline

### CODE REVIEW

File: js/components/activities/timeline.vue [hunk 1]
```diff
@@ -39,6 +39,10 @@ import moment from 'moment';
 import ActivityPage from 'models/activities';
 import URLs from 'urls';
 
+function actor(activity) {
```
Comment (davidbgk) [inline js/components/activities/timeline.vue]: Oh nice trick :)

**Comment by davidbgk** [comment]
r+wc

File: js/components/activities/timeline.vue (hunk 1 above)
Comment (noirbizarre) [inline js/components/activities/timeline.vue]: ;)
Which `v-if`, these one ? https://github.com/etalab/udata/blob/timeline-adjustments/js/components/activities/timeline.vue#L17-L18

Comment (davidbgk) [inline js/components/activities/timeline.vue]: Yes, and now I realize that the label is not the same… nevermind.

**Pull request merged**